app/controllers/posts_controller.rb

```diff
@@ -346,25 +346,23 @@ def mark_hidden
   end
 
   def change_status
-    begin
-      new_status = Post::Status.get_status(params[:status].upcase)
```

Comment:
this is where we were using get_status, I think. I'd grep the code for any other instances but you might have obsoleted it yeah